toolkit/smart-contracts/commands/src/reserve.rs

```diff
@@ -91,18 +91,25 @@ pub struct CreateReserveCmd {
 	#[arg(long)]
 	/// Reserve token asset id encoded in form <policy_id_hex>.<asset_name_hex>.
 	token: AssetId,
+	#[arg(long, default_value = "1")]
+	/// Amount of illiquid circulation supply authority tokens to mint.
```

Comment:
Naming discussion.
1. I think tokens are only the means to keep some minimal number of UTXOs at ICS Validator", so the parameter  could be called "ics_initial_utxos_amount".

2. I don't know how these tokens are "authority tokens" in any way (eventual renaming should go into PCSC as well). For me the that are some "protected" tokens, or "validator tokens" (because they can't leave the validator). 

This thing is hard to name, because the concept of why we need these special tokens and utxos isn't simple as well.